util/mount.fuse.c

```diff
@@ -441,5 +441,9 @@ int main(int argc, char *argv[])
 	execl("/bin/sh", "/bin/sh", "-c", command, NULL);
 	fprintf(stderr, "%s: failed to execute /bin/sh: %s\n", progname,
 		strerror(errno));
+	
+	if (pass_fuse_fd)
+		close(fuse_fd);
```

Comment:
As far as I can see, fuse_fd isn't even defined here. Have you tried if this compiles?

Reply:
I'm sorry for no compile. Now I move definition of fuse_fd to the begin of main.